instrument/IN5_Definition.xml

```diff
@@ -1,722 +1,712 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- For help on the notation used to specify an Instrument Definition File see http://www.mantidproject.org/IDF -->
-<instrument xmlns="http://www.mantidproject.org/IDF/1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mantidproject.org/IDF/1.0 Schema/IDFSchema.xsd" name="IN5" valid-from="1900-01-31 23:59:59"
-valid-to="2100-01-31 23:59:59" last-modified="2018-03-27 11:19:26">
-  <!-- Author: ricardo.leal@ill.fr -->
+<?xml version='1.0' encoding='ASCII'?>
```

Comment:
Can we stick to one quote style.

Reply:
Interesting. The file is automatically generated using `lxml` so I'd guess the same issue appears in other IDFs as well. I'll see if I can fix the IDF generator helpers in `mantidgeometry` in this regard.

Reply:
Any I'd worked on in the past all used double quotes but looking again I see there are a number that use single quotes (although just for the `xml` tag, nowhere else).

In that case I'm fine with this the way it is and standardizing can be a separate issue.

Reply:
The quote style is set in stone when calling `lxml.etree.tostring()` or at least I cannot find a way to change it. We could change the quotes manually, but I am afraid this will be lost next time someone else reruns the generator script. Would it be OK to leave the quotes as is?

Reply:
So it is OK, then. Sorry, missed your last comment here.

Reply:
Yep, fine with me.